services/basket/src/main/java/org/cryptimeleon/incentive/services/basket/BasketService.java

```diff
@@ -19,46 +18,51 @@
 public class BasketService {
 
     private final HashMap<UUID, Basket> basketMap;
-    private final ArrayList<Item> items;
+    private final List<RewardItem> rewardItems;
```

Comment:
In an ideal world, we would of course have a database behind this but I doubt that we will have time for this, especially considering that it will make no difference for the presentation of the project